### ISSUE TITLE
[Bug] Dap does not recognize launch.json

### ISSUE THREAD

**Issue by chriztheanvill** [nvim-telescope/telescope-project.nvim#72]
Hi.
I am using dap, and does not recognize "vscode/launch.json" file if I use Telescope projects. But, if I load the source and try dap, it recognize "vscode/launch.json".
--(02:59:04) - (14 files: 6.3Mb)$ tree -P .
.
├── .vscode
│   ├── launch.json
├── src
├── assets
│   ├── config
│   ├── fonts
│   ├── images
│   ├── music
│   ├── scripts
│   ├── sounds
│   └── tilemaps
├── build
├── google_test
├── My_Cmake
├── SH_files

**Comment by matu3ba** [comment]
Can you print your cwd?

**Comment by chriztheanvill** [comment]
> Can you print your cwd inside neovim, ie with `:pwd`? Check how dap finds the launch.json.

Yes, it shows the correct path with pwd, but does not recognize the launch.json file. I asked also in the github of dap, and they told me that, project should auto reload the path.

**Comment by matu3ba** [comment]
> project should auto reload the path

This is logically inconsistent. Probably they mean that you should reload you lua config from that path, which is abit nicer with plenary. https://www.reddit.com/r/neovim/comments/jxub94/reload_lua_config/

Can you ask them to give you a stripped down example? I did not configure this yet.
